awx/main/tasks.py

```diff
@@ -1263,6 +1265,7 @@ def run(self, pk, **kwargs):
                     params.pop('inventory'),
                     os.path.join(private_data_dir, 'inventory')
                 )
+                params['envvars'].pop('HOME')
```

Comment:
Good find.

Reply:
Haha. This one was a fun one to track down. This is the wrong `HOME` altogether! This is passed through to the environment on the isolated node but it's actually read / set from the environment on the *controller*.

Reply:
We enforce the same dir/home structure on the iso nodes as on non-iso full-tower nodes. Thus, we know `/var/lib/awx` to be the home dir on an ISO node.

So I would say, it is fine to strip this when the iso node is this new container, but leave `HOME` defined for legacy isolated.